demos/markdown/cli-basics.didact.md

```diff
@@ -0,0 +1,79 @@
+
+### The `pwd` command
+
+First, let's see what your current directory is. Type `pwd` ([▶️](didact://?commandId=vscode.didact.sendNamedTerminalAStringNoLF&text=newTerminal$$pwd)) at the command line and hit Enter.
+
+You should see something like `/home` followed by your user name and the directory of your VS Code workspace. For example: `/home/myuser/demo`. 
+
+### The `mkdir` command
+
+Let's create a sub-directory, or a directory within our current directory, named `practice` with the command `mkdir practice`. ([▶️](didact://?commandId=vscode.didact.sendNamedTerminalAStringNoLF&text=newTerminal$$mkdir%20practice))
+
+### The `ls` command
+
+To see what files and sub-directories are in a particular directory, you can use the `ls` command. To verify that our `practice` directory was created successfully, let's type `ls` ([▶️](didact://?commandId=vscode.didact.sendNamedTerminalAStringNoLF&text=newTerminal$$ls)) in our practice directory and press Enter.
+
+You should see that there's now a directory called `practice` in this directory.
+
+### The `cd` command
+
+Now, to move into that directory, let's type `cd practice`. ([▶️](didact://?commandId=vscode.didact.sendNamedTerminalAStringNoLF&text=newTerminal$$cd%20practice))
+
+Remember that `pwd` ([▶️](didact://?commandId=vscode.didact.sendNamedTerminalAStringNoLF&text=newTerminal$$pwd)) command from earlier? Let's try that again and you should see `/practice` tacked on at the end.
+
+We can also run the `ls` ([▶️](didact://?commandId=vscode.didact.sendNamedTerminalAStringNoLF&text=newTerminal$$ls)) command to see if there's anything in our new directory. It's empty, so let's create a file to make it less lonely!
+
+### The `touch` command
+
+Next we can use the `touch` command to create a new, empty file. Type `touch example.txt` ([▶️](didact://?commandId=vscode.didact.sendNamedTerminalAStringNoLF&text=newTerminal$$touch%20example.txt)) and press Enter to create a file named `example.txt`.
+
+If we run the `ls` ([▶️](didact://?commandId=vscode.didact.sendNamedTerminalAStringNoLF&text=newTerminal$$ls)) command again, you should see your new file!
+
+## Reading the Contents of a File
+
+Creating empty files isn't all that fun, but there are other commands we can use to put some text into them and view them as well!
+
+Remember the `echo` command? Let's use it in a new way!
+
+Type `echo "some sample text" > sample.txt`, ([▶️](didact://?commandId=vscode.didact.sendNamedTerminalAStringNoLF&text=newTerminal$$echo%20%22some%20sample%20text%22%20%3E%20sample.txt)) and hit Enter. This creates a new file and puts some text into it from the start.
+
+We can then use the `cat` command to show the file contents. Type `cat sample.txt` ([▶️](didact://?commandId=vscode.didact.sendNamedTerminalAStringNoLF&text=newTerminal$$%60cat%20sample.txt)) to print out the contents of our file. You should see `Some example text` show up on the very next line!
```

Comment:
```suggestion
We can then use the `cat` command to show the file contents. Type `cat sample.txt` ([▶️](didact://?commandId=vscode.didact.sendNamedTerminalAStringNoLF&text=newTerminal$$cat%20sample.txt)) to print out the contents of our file. You should see `Some example text` show up on the very next line!
```

otherwise it is nto working because there i an extra backtick and hittign enter is askign for more inputs on new line